lib/sys.js

```diff
@@ -21,6 +21,14 @@
 
 'use strict';
 
+var util = require('util');
+
 // the sys module was renamed to 'util'.
 // this shim remains to keep old programs working.
-module.exports = require('util');
+// sys is deprecated and shouldn't be used
+
+var setExports = util.deprecate(function() {
+  module.exports = util;
+}, 'sys is deprecated, use util instead');
```

Comment:
Small nit, but could you change this to:

`sys is deprecated. Use util instead.` (Ref: https://github.com/iojs/io.js/pull/166#discussion_r21848286)